sky/serve/infra_providers.py

```diff
@@ -477,6 +486,12 @@ def _teardown_cluster(self,
         info = self.replica_info[cluster_name]
         info.status_property.sky_down_status = ProcessStatus.RUNNING
 
+    def _recover_from_preemption(self, cluster_name: str) -> None:
+        # TODO(tgriggs): Hook in policy strategy options here.
+        logger.info(f'Beginning recovery for preempted cluster {cluster_name}.')
+        self.replica_info[cluster_name].status_property.preempted = True
+        self._teardown_cluster(cluster_name, sync_down_logs=False)
```

Comment:
Add a comment for reasons of `sync_down_logs=False` here? (sth like `cluster_status != UP` )

Reply:
Done, thanks.